--- libffado-2.4.2/SConstruct.orig	2019-09-21 08:40:33.000000000 +0200
+++ libffado-2.4.2/SConstruct	2020-03-10 05:40:06.349185946 +0100
@@ -836,6 +836,7 @@
             print("Doing a 32-bit %s build for %s" % (cpuinfo.machine, cpuinfo.model_name))
             if cpuinfo.machine == 'x86_64':
                 machineflags = { 'CXXFLAGS' : ['-mx32'] }
+                needs_fPIC = True
             else:
                 machineflags = { 'CXXFLAGS' : ['-m32'] }
         else:
