--- libffado-2.2.1/tests/test-enhanced-mixer.cpp.orig	2008-05-29 10:24:21.000000000 +0200
+++ libffado-2.2.1/tests/test-enhanced-mixer.cpp	2016-02-16 18:02:11.361870351 +0100
@@ -77,7 +77,7 @@
         exit(0);
     }
 
-    int errno = 0;
+    errno = 0;
     char* tail;
     int node_id = strtol( argv[1], &tail, 0 );
     int fb_id   = strtol( argv[2], &tail, 0 );
--- libffado-2.2.1/tests/test-volume.cpp.orig	2008-05-05 23:29:40.000000000 +0200
+++ libffado-2.2.1/tests/test-volume.cpp	2016-02-16 18:12:32.575177616 +0100
@@ -127,7 +127,7 @@
         exit(0);
     }
 
-    int errno = 0;
+    errno = 0;
     char* tail;
     int node_id = strtol( argv[1], &tail, 0 );
     int fb_id   = strtol( argv[2], &tail, 0 );
